draft-ietf-jose-hpke-encrypt.md

```diff
@@ -300,6 +303,17 @@ this document, the valid combinations of the KEM, "kty" and "crv" are shown in
 {: #ciphersuite-kty-crv title="JWK Types and Curves for JOSE-HPKE Ciphersuites"}
 
 
+## Key Usage Guidelines for JOSE-HPKE
+
+To ensure predictable key usage within JOSE-HPKE, the following restrictions and guidelines are introduced:
+
+1. **New Key Use Values**
+   The following values are registered in the "JSON Web Key Use" registry to explicitly identify the roles of keys in HPKE operations:
+   - **HPKEauth:** A key intended for use in the sender role of HPKE operations, performing encryption and key encapsulation.
+   - **HPKErecv:** A key intended for use in the receiver role of HPKE operations, performing decryption and key decapsulation.
+
+These values allow implementations to explicitly track and enforce role-specific key usage in HPKE operations and prevent key reuse with other cryptographic algorithms.
+
```

Comment:
```suggestion
```

The correct key use value for any encryption operation is "enc", which is already defined.

Reply:
The new key use values are introduced to address mandatory security requirements outlined in the Key Management Section. Enforcing these rules would not be feasible without defining these values.

Reply:
lets make sure to mirror this in COSE, or leave it out.

Reply:
Unfortunately, COSE does not currently have "use" (only the broken key_ops).

Might be worth adding "use" to COSE (like "typ" was recently added).